src/components/button/style/button.scss

```diff
@@ -0,0 +1,17 @@
+@import '../../../Sass/App.scss';
+
+.ui.blue.basic.button{
+    color: #2F80ED !important;
+    background: #E0ECFD !important;
```

Comment:
yeah i am planning also on that when im finished on the components and refactor all the css. Anyway since i will be trying the styled-component this will be re updated :)